### ISSUE TITLE
8.0.0 - Minimize / Minify / Uglify issue - table does not work

### ISSUE THREAD

**Issue by AndreyKomin** [handsontable/handsontable#7207]
### Description
The table stops working if I build it in Production mode within a vue-cli project.
It seems a minification issue - if I disable `minimize` option in `vue.config.js` then it works.
![image](https://user-images.githubusercontent.com/7191525/90779690-cc975480-e306-11ea-99cf-76c32fe9ab1f.png)

It worked correctly in the 7.4.2 ver.

### Steps to reproduce

1. Unzip the `Archive.zip` (attached)
2. `npm install`
3. `npm run build && npm run start` (http://localhost:8080)
then you'll see this error:
![image](https://user-images.githubusercontent.com/7191525/90778003-c7d1a100-e304-11ea-92ff-e9986b5459f5.png)

Note: If you run it in a dev mode it would work (`npm run serve`)

### Demo
[Archive.zip](https://github.com/handsontable/handsontable/files/5103487/Archive.zip)

### Your environment
* Handsontable version: 8.0.0
* handsontable/vue version: 5.0.1
* Browser Name and version: Chrome 84
* Operating System: macOS Catalina

**Comment by AMBudnik** [comment]
@AndreyKomin 

Sorry for keeping you waiting. I have replicated the error in the console on my Windows 10.

I've removed Handsontable and the wrapper from the dependency tree and reloaded it back again (using HOT 8.0.0 / Vue wrapper 5.0.1) but got the same issue. Once I loaded HOT 7.4.2 and Vue wrapper 4.1.1 problem was gone.

One thing remains a mystery though. I wasn't able to find the `minimize` option in the project.
